pkg/mcs/tso/server/server.go

```diff
@@ -71,9 +71,10 @@ const (
 	tsoSvcRootPathFormat = msServiceRootPath + "/%d/" + mcsutils.TSOServiceName
 
 	// maxRetryTimesWaitAPIService is the max retry times for initializing the cluster ID.
-	maxRetryTimesWaitAPIService = 60
+	maxRetryTimesWaitAPIService = 360
 	// retryIntervalWaitAPIService is the interval to retry.
-	retryIntervalWaitAPIService = 3 * time.Second
+	// Note: the interval must be less than the timeout of tidb and tikv, which is 2s by default in tikv.
+	retryIntervalWaitAPIService = 500 * time.Millisecond
```

Comment:
Will it print more logs?

Reply:
the log level is only debug.

Reply:
https://github.com/tikv/pd/blob/19f5bd531d3f09de6cda6c4ab0391eae79b3fd96/pkg/mcs/tso/server/server.go#L551

Reply:
No, there is a warning log instead.